### ISSUE TITLE
`build!` macro doesn't have the same syntax as use statements

### ISSUE THREAD

**Issue by Awfa** [microsoft/windows-rs#699]
The `build!` macro docs say this:
> These follow the same convention as Rust use paths.

However, this fails to compile:
```rust
windows::build!(
    Windows::Win32::{
        Debug::GetLastError
    }
)
```
with the following error message:
```
error: proc macro panicked
  --> src\rust\winapi_wrapper\build.rs:30:5
   |
30 | /     windows::build!(
31 | |         Windows::Win32::{
32 | |             Debug::GetLastError
33 | |         }
34 | |     )
   | |_____^
   |
   = help: message: Could not find namespace `Windows.Win32`
```

The following *does* compile however:
```rust
windows::build!(
    Windows::Win32::Debug::{
        GetLastError
    }
)
```

I expect the first one to compile because that's how Rust use paths work.
The error message about not finding the namespace is confusing too.

For context, I ran into this issue when trying to use `build!` like this:
```rust
windows::build!(
    Windows::Win32::{
        Debug::GetLastError,
        Etw::{
            EventRegister,
            EventUnregister,
            EventWrite,
            EVENT_DATA_DESCRIPTOR,
            EVENT_DESCRIPTOR
        },
        ProcessStatus::K32EnumProcesses,
        WindowsProgramming::CloseHandle,
        WindowsEventLog::{
            EvtClose,
            EvtOpenPublisherMetadata,
            EVT_HANDLE
        },
        SystemServices::{
            ERROR_INSUFFICIENT_BUFFER,
            OpenProcess,
            PSTR,
            QueryFullProcessImageNameA,
            TerminateProcess,
        },
    }
)
```

**Comment by kennykerr** [comment]
Glad you found a workaround. Yes, this doesn't work very smoothly today. I'm planning to improve the build parsing soon as the current implementation has a few drawbacks.

**Comment by kennykerr** [comment]
This should be fixed as of #773 - let me know if you notice any further issues.
